packages/eslint-config-react-a11y/rules/jsx-a11y.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+module.exports = {
+    'rules': {
+        // Enforce emojis are wrapped in and provide screen reader access.
+        // https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/HEAD/docs/rules/accessible-emoji.md
```

Comment:
Can we remove all these URLs in favor of just one at the top? See https://github.com/moxystudio/eslint-config/blob/master/packages/eslint-config-react/rules/react-hooks.js#L1